product_docs/docs/pgd/5/cli/installing_cli.mdx

```diff
@@ -37,11 +37,10 @@ cluster:
   - "host=bdr-c1 port=5432 dbname=bdrdb user=postgres "
 ```
 
-The `pgd-config.yml`, is located in the `/etc/edb` directory, by default. The PGD CLI searches for `pgd-config.yml` in the following locations (precedence order - higher to lower):
+The `pgd-cli-config.yml` (in v1 it was `pgd-config.yml`), is located in the `/etc/edb/pgd-cli` directory, by default. The PGD CLI searches for `pgd-cli-config.yml` in the following locations (precedence order - higher to lower):
```

Comment:
Revise to:
By default, `pgd-cli-config.yml` is located in the `/etc/edb/pgd-cli` directory. (in v1, the file was named`pgd-config.yml`.) The PGD CLI searches for `pgd-cli-config.yml` in the following locations. Precedence order is higher to lower.

Could you just say high to low?

Reply:
sure, please check this (also added "default location was `/etc/edb`") -

```
By default, `pgd-cli-config.yml` is located in the `/etc/edb/pgd-cli` directory. In v1, the file was named `pgd-config.yml` and default location was `/etc/edb`. The PGD CLI searches for `pgd-cli-config.yml` in the following locations. Precedence order is high to low.

  1. `/etc/edb/pgd-cli` (default)
  2. `$HOME/.edb/pgd-cli`

```